components/helpers/SiteMessages.php

```diff
@@ -0,0 +1,33 @@
+<?php
+//******************************************************************************
+//                                       SiteMessages.php
+// SILEX-PHIS
+// Copyright © INRA 2018
+// Creation date: 10 Sept, 2018
+// Contact: arnaud.charleroy@inra.fr, anne.tireau@inra.fr, pascal.neveu@inra.fr
+//******************************************************************************
+
+namespace app\components\helpers;
+
+/**
+ * SiteMessages a class which defines errors messages
+ * @author Arnaud Charleroy <arnaud.charleroy@inra.fr>
+ */
+class SiteMessages {
+    // Page routes
+    CONST SITE_ERROR_PAGE_ROUTE = '/site/error';
```

Comment:
In php keyword are lowercase, please could you change.
We'll add this in coding style.

Reply:
Done